linode/manager
# feat: [M3-8570] - Finish sunsetting Gravatar

## Description 📝
This is a follow up from M3-8158 (#10859), which informed users with Gravatars of a sunset date of Sept 30 and transitioned users not currently using Gravatars to our new initial and color avatar icon.

This PR will remove Gravatar support for users who still have it on the communicated date.

## Changes  🔄
- Deletes a bunch of Gravatar-related components and utils
- Uses our Avatar component only
- Changes the username for system events or opened support tickets from "Linode" to "Akamai"

## 📸 Preview
| Header | Header |
|--------|--------|
| ![Screenshot 2024-09-16 at 9 16 33 AM](https://github.com/user-attachments/assets/b5b245d4-243b-4265-b70f-1cb0b03896c8) | ![Screenshot 2024-09-16 at 9 15 39 AM](https://github.com/user-attachments/assets/7307d007-838e-4ea1-9259-d526b0203f6e) |
| ![Screenshot 2024-09-16 at 9 16 22 AM](https://github.com/user-attachments/assets/fd405201-c5e9-4de3-83c4-129bacfa7385) | ![Screenshot 2024-09-16 at 9 15 48 AM](https://github.com/user-attachments/assets/428c1330-417f-4a78-85dc-cbea41d48837) |
| ![Opt2](https://github.com/user-attachments/assets/38a06c5d-dea7-41b3-b150-48e5a9e46348) | ![Opt1](https://github.com/user-attachments/assets/9b0cdf99-333f-48f6-babd-9f6098c07e82)|

## Target release date 🗓️

9/30/24

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Log in as an account with a Gravatar

### Verification steps
(How to verify changes)
- Go to the following locations in the app where we render avatars and confirm that you do not see your Gravatar anywhere:
   -  Top Menu
   - Notification Center Event
   - Events Landing > Event Table Row
   - User Landing > User Table Row
   - User SSH Key Panel
   - Profile Display
   - Support Ticket Details (test this in dev with a dummy support ticket you comment on)
- Confirm tests pass

## As an Author I have considered 🤔

*Check all that apply*
